config.cc

```diff
@@ -38,12 +39,17 @@ namespace
       char const * pathInHome = "GoldenDict";
       result = QDir::fromNativeSeparators( QString::fromWCharArray( _wgetenv( L"APPDATA" ) ) );
     #else
-      char const * pathInHome = ".goldendict";
+      QString configPath = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation);
+      if (configPath.isEmpty()) {
+          configPath = ".config";
+      }
+      configPath += "/goldendict";
+      std::string pathInHome = configPath.toStdString();
```

Comment:
You are changing the configuration location for all users here. Everyone will suddenly lose their settings and have to copy them into the new location. Imagine the amount of noise on the issue tracker. If you feel strongly about this config file location, you can just create a symbolic link *~/.goldendict* pointing to *~/.config/goldendict* on your system. I don't recall anyone else complaining about the current location.